python/pyarrow/array.pxi

```diff
@@ -1778,6 +1778,70 @@ cdef class Array(_PandasConvertible):
 
         return pyarrow_wrap_array(array)
 
+    def _export_to_c_device(self, out_ptr, out_schema_ptr=0):
```

Comment:
`out_schema_ptr=None` would feel slightly more Pythonic IMHO, though that's debatable.

Reply:
I would propose to leave this as is, to keep it consistent with the other `_export_to_c` definitions (and the `_as_c_pointer` helper also requires an integer at the moment)